src/main/resources/templates/register.html

```diff
@@ -2,28 +2,21 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org"
       xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity3">
 <head>
-    <script th:if="${failure}" src="js/register.js" type="application/javascript"></script>
     <th:block th:include="fragments/header :: head (title='DigiWill - Register')"></th:block>
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
 </head>
 <body onload="onLoad()">
 <div th:include="fragments/header :: header"></div>
 
-<script th:if="${failure}">
+<script th:if="${hasToast}">
```

Comment:
Hatte gedacht, das hier mit in den footer, dass es einfach auf jeder seite so funktioniert

Reply:
Du hast halt vielleicht nicht auf jeder Seite ein hasToast im Thymeleaf. Du kannst auf jeder Seite mit launchToast(errorText) ein Errorfenster öffnen lassen, fand ich jetzt besser

Reply:
Achso, mh, kann auch sinn machen. Wieso ist das dann an der stelle hie rnicht so? Hier wird ja das element nochmal manuell geholt und so

Reply:
Änder ich schnell, dann machts das auch so

Reply:
Funktioniert das so oder würde es mehr sinn machen oben beim body statt dem onLoad gleich mit ner thymeleaf abfrage den launchToast() aufzurufen?